docs/samples/label/innerChart.md

```diff
@@ -0,0 +1,152 @@
+# Inner chart
+
+```js chart-editor
+// <block:setup:5>
+const DATA_COUNT = 12;
+const MIN = 0;
+const MAX = 100;
+
+const innerChart = Utils.getChart();
+const innerDataset = innerChart.data.datasets[0];
+
+const numberCfg = {count: DATA_COUNT, min: MIN, max: MAX};
+
+const data = {
+  labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+  datasets: [{
+    data: Utils.numbers(numberCfg),
+    borderColor: innerDataset.backgroundColor[0]
+  }, {
+    data: Utils.numbers(numberCfg),
+    borderColor: innerDataset.backgroundColor[1],
+    hidden: true
+  }, {
+    data: Utils.numbers(numberCfg),
+    borderColor: innerDataset.backgroundColor[2],
+    hidden: true
```

Comment:
Add `backgroundColor: 'transparent'` here and remove `autocolors: false` in the plugin. Makes it a bit easyer for people who dont know much about other chart.js plugins.

Reply:
done!